src/middleware/jsx/index.ts

```diff
@@ -54,6 +54,11 @@ const jsxFn = (
       continue
     } else if (v === null || v === undefined) {
       continue
+    } else if (['checked', 'selected'].includes(propsKeys[i]) && v === false) {
```

Comment:
I've investigated HTML specs. There are other boolean attributes, `disabled`, `readonly` and`multiple`.
See: https://stackoverflow.com/questions/706384/boolean-html-attributes
Lets' add them.